src/regedit/register.jl

```diff
@@ -169,11 +96,29 @@ function check_version(existing::Vector{VersionNumber}, ver::VersionNumber)
 end
 
 """
-Register the package at `package_repo` / `tree_spect` in `registry`.
+    register(package_repo, pkg, tree_hash; registry, registry_deps, push, gitconfig)
+
+Register the package at `package_repo` / `tree_hash` in `registry`.
+Returns a `RegEdit.RegBranch` which contains information about the registration and/or any
+errors or warnings that occurred.
+
+# Arguments
+
+* `package_repo::String`: the git repository URL for the package to be registered
+* `pkg::Pkt.Types.Project`: the parsed Project.toml file for the package to be registered
+* `tree_hash::String`: the tree hash (not commit hash) of the package revision to be registered
+
+# Keyword Arguments
+
+* `registry::String=$DEFAULT_REGISTRY_URL`: the git repository URL for the registry
```

Comment:
```suggestion
* `registry::String="$DEFAULT_REGISTRY_URL"`: the git repository URL for the registry
```

Reply:
I meant to do this as a suggestion, and not actually commit to your branch... Sorry :grimacing:. GitHub was a bit confusing there.

Reply:
oh well, I'd accept it anyway!